src/System.Diagnostics.Metrics.Temp/Distribution.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Diagnostics.Metrics
+{
+    public class Distribution<T> : MeterInstrument<T> where T : unmanaged
```

Comment:
There are many places not updated to the latest API spec:
1. Instrument names (Distribution -> Histogram, Gauge -> ObservableGauge).
2. Label - Attribute (or Tag?)